src/zeit/content/volume/testing.py

```diff
@@ -18,10 +18,20 @@
 </product-config>
 """.format(here=pkg_resources.resource_filename(__name__, '.'))
 
+# Config duplication, because z.c.article.testing can't be imported due to
```

Comment:
Merkwürdig, ich habe so vage in Erinnerung, dass ich die article product conf auch für die tests brauche. Jetzt gelöscht und siehe da Tests laufen auch so...